x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/string/Space.java

```diff
@@ -116,7 +117,7 @@ public ExpressionEvaluator.Factory toEvaluator(ToEvaluator toEvaluator) {
             Object folded = field.fold(toEvaluator.foldCtx());
             if (folded instanceof Integer num) {
                 checkNumber(num);
-                return toEvaluator.apply(new Literal(source(), " ".repeat(num), KEYWORD));
+                return toEvaluator.apply(new Literal(source(), BytesRefs.toBytesRef(" ".repeat(num)), KEYWORD));
```

Comment:
This seems to repeat a lot, I wonder if it is worth introducing:
```
pusblic static Literal.keyword(Source source, String literal) {
    return new Literal(source, BytesRefs.toBytesRef(literal), KEYWORD);
}
```
to simplify construction of string literals?

Reply:
👍 I think it makes sense